<launch>
    <!-- Run our dynamic scene model server node + simulation -->
	<include file="$(find srs_env_model)/launch/but_dynmodel_majkl2.launch"/>
    
    <!-- Run interactive markers server -->
        <node pkg="srs_interaction_primitives" type="interaction_primitives_service_server" name="Interactive server"/>
</launch>

